### PR TITLE
add manpage

### PR DESCRIPTION
**Pull request by Konfekt**
addresses https://github.com/euank/pazi/issues/81

### CODE REVIEW

File: pazi.1 [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+view      View the frecency database
+\f[R]
+.fi
+.SH DESCRIPTION
+.PP
+Pazi is an autojump utility.
+That is to say, pazi remembers visited directories in the past and makes
+it easier to get back to them.
+.PP
+There are several autojump utilities, including fasd (or a better
```
Comment (euank) [inline pazi.1:56]: The '(or a better maintained fork)' bit doesn't really work without the hrefs.

I think the man page might read more cleanly without this paragraph entirely.

File: pazi.1 [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+.PP
+OPTIONS:
+.IP
+.nf
+\f[C]
+    --debug <debug>    print debug information to stderr [env: PAZI_DEBUG=]
+\f[R]
+.fi
+.PP
+SUBCOMMANDS: edit Edit the frecency database
```
Comment (euank) [inline pazi.1:37]: I think `edit` is meant to be a little further down, inside the `\f[C]`?

**Comment by euank** [comment]
Oh, there is one bikeshed; I don't think this belongs at the root of the repo, but I'm not sure where it should go.

Maybe `packaging/man/pazi.1`?

**Comment by Konfekt** [comment]
Thank you for reviewing! The latest commit applies your suggestions.

**Pull request merged**